docs/auditing.md

```diff
@@ -0,0 +1,9 @@
+# Whitehall Content Auditing
```

Comment:
Just thinking about findability in the dev docs (all the documentation in the docs folder is imported into the dev docs), is it worth putting Audit trail in the title of this page?

Reply:
Good shout